### DISCUSSION TITLE
Use of MIDI CC Data

### DISCUSSION THREAD

**Discussion by clementc2411** [mik3y/pymidi#21] [Q&A]
Hello, first of all many thanks for your work! I have used pyMidi to control a homemade led tube wirelessly with a RPI0 (see it working [here](https://youtube.com/shorts/az3mBKrKfds?feature=share)).
It works well with NOTE ON/OFF events but I am now interested to control the leds via MIDI CC. This would allow me to easily program color changes, brightness variations directly in my DAW.
Would anyone have a code example on how to handle the MIDI CC data ?

**Comment by mik3y** [comment]
Hey there, it's great to hear you built something fun with the library! I'll look into this - no promises on implementation, but a quick glance at the protocol & looks straightforward. 

Filed an issue to track it here: https://github.com/mik3y/pymidi/issues/22
